scripts/release-orchestrator.mjs

```
@@ -0,0 +1,690 @@
+  const [command, ...rest] = args;
+  const options = parseOptions(rest);
+  if (command === "prepare") return prepareRelease(options);
+  if (command === "publish") return publishRelease(options);
+  throw new ReleaseError(
+    "Usage: release-orchestrator.mjs <prepare|publish> [--bootstrap] [--tag <tag>] [--yes]",
+  );
+}
+
+if (process.argv[1] && resolve(process.argv[1]) === fileURLToPath(import.meta.url)) {
+  main().catch((error) => {
+    console.error(error instanceof Error ? error.message : error);
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: clear, info.

(no-console)